Makefile

```diff
@@ -66,6 +66,10 @@ clean:
 # generate project
 #################################################################
 
+overwrite:
+	rm -rf $(COOKIECUTTER_OUTPUT_DIR)/behave* $(COOKIECUTTER_OUTPUT_DIR)/features
```

Comment:
Add a comment to explain that this is a bug workaround

Reply:
done